fmriprep/workflows/fieldmap/unwarp.py

```diff
@@ -126,6 +127,10 @@ def init_sdc_unwarp_wf(reportlets_dir, omp_nthreads, fmap_bspline,
     # Fieldmap to rads and then to voxels (VSM - voxel shift map)
     torads = pe.Node(niu.Function(function=_hz2rads), name='torads')
 
+    get_ees = pe.Node(niu.Function(
+        function=_get_ees, input_names=['in_meta', 'in_file'],
```

Comment:
I'd prefer not to have `input_names`, if it's all the same to you, but wouldn't argue over it if you prefer to keep them in.